KsApi/models/Comment.swift

```diff
@@ -1,15 +1,85 @@
 import Foundation
 
-public struct Comment: Decodable, Equatable {
+/**
+ FIXME: Some of the `Comment`'s properties does not represent `GraphComment` mapping but rather the expected structure.
+ Some of these properties might need to be updated once `GraphComment` has the needed properties for `Comment`.
+ */
+
+public struct Comment {
   public var author: Author
+  public var authorBadges: [AuthorBadge]?
   public var body: String
+  public let createdAt: TimeInterval
+  public let deletedAt: TimeInterval?
   public var id: String
+  public var isDeleted: Bool
+  public var isFailed: Bool = false
   public var uid: Int
   public var replyCount: Int
 
+  /// return the first `authorBadges`, if nil  return `.backer`
+  public var authorBadge: AuthorBadge {
+    return self.authorBadges?.first ?? .backer
```

Comment:
I think this is fine but should run it by Paulo so Product is aware of this convention

Reply:
Resolving since we are continuing this on the following thread:
https://kickstarter.slack.com/archives/CG55BBBBL/p1621968220061600